README.md

```diff
@@ -80,6 +80,39 @@ The available configuration presets to choose from:
 
 It's important to note that you can still override any of the preset rule definitions in your configuration. Think of these presets as convenience "defaults" that can still be customized.
 
+### Flat Config
+
+If you're using ESLint's newer flat config format, you can use the plugin's flat configuration preset:
+
+```js
+// eslint.config.js
+import arrows from "@getify/eslint-plugin-proper-arrows";
```

Comment:
maybe it would be a little more proper to name this import name `properArrows` since that's the name of the plugin? Same with the other instances here in your additions.